incubator/zookeeper/templates/ss.yaml

```diff
@@ -142,11 +140,7 @@ spec:
   - metadata:
       name: datadir
       annotations:
-        {{- if .Values.StorageClass}}
-        volume.beta.kubernetes.io/storage-class: {{.Values.StorageClass | quote}}
-        {{- else}}
-        volume.alpha.kubernetes.io/storage-class: "default"
-        {{- end}} 
+        volume.beta.kubernetes.io/storage-class: {{default "standard" .Values.StorageClass | quote}}
```

Comment:
Fair enough, I can close the PR in light of the work done in the other.  As a side note, is there ever a reason to use the `helm.sh/created` type annotation on these resources?  It seems like an anit-pattern for upgrading purposes.